docs/dev/testing.md

```diff
@@ -0,0 +1,42 @@
+# Testing Pull Requests 
+
+One way to help pyDARN get developed faster and smoother is by testing [Pull Requests](https://github.com/SuperDARN/pydarn/pulls). 
+This can take fifteen minutes or less to do, helps the developer merge their code,
+and allows you to be on the author list!
+
+Every contribution helps! 
+
+## Setting up the testing environment
+
+If you haven't cloned pyDARN before this will get you the correct branch installed to just test:
+`pip install git+https://github.com/superdarn/pydarn@<branchname>`
```

Comment:
```suggestion
If you haven't cloned pyDARN before this will get you the correct branch installed to just test:
`pip install git+https://github.com/superdarn/pydarn@<branchname>`
Make sure to install any new instances of pyDARN, or branches that are not master in a virtual environment to avoid mixing pyDARN versions! 
```